Source/Core/Localization.swift

```diff
@@ -8,64 +8,56 @@
 
 import UIKit
 
-public enum Language: String {
-    case english = "en"
-    case hebrew = "he"
-    case russian = "ru"
-}
-
 public class Localization: NSObject {
 
     private enum Constants {
         static let userLanguageSavedKey = "UserLanguageSavedKey"
     }
 
-    private static var defaultLanguage: Language!
+    private static var defaultLanguage: String!
     private static var usePreferredLanguage: Bool!
 
-    public static func setup(defaultLanguage: Language = .english, usePreferredLanguage: Bool = true) {
+    public static func setup(defaultLanguage: String = "en", usePreferredLanguage: Bool = true) {
```

Comment:
I should be required.

Reply:
So, we use "Base" language instead of default